src/devices/MettlerToledo/Exceptions/BalanceOutOfRangeException.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
```

Comment:
Missing licensing headers everywhere. Please see on other bindings. It needs to be added everywhere.